.cargo/linux_runner_wrapper.sh

```diff
@@ -2,7 +2,7 @@
 
 binary_name=`basename $1`
 
-if [[ $binary_name =~ ^(north|tests-[a-z0-9]{16})$ ]]; then
+if [[ $binary_name =~ ^(north|integration_tests-[a-z0-9]{16})$ ]]; then
```

Comment:
Is it `integration_test` or `north_test`?

Why not just sticking to `tests`?

Reply:
it is `integration_test`. I was looking for examples on how people implement this kind of integration tests. This seems to be the pattern that most projects use.

The reason for the rename is that `tests` has a special treatment as it is the place where integration tests are supposed to go. The content of the `tests` folder is compiled separately from the crate itself. The crate `north_tests` will contain tests utils in general (including possible tests for those features) while the `tests` folder will contain the actual integration tests that's compiled separately to all the crates, including `north_tests`.

Maybe `northstar_tests` is a better name.